src/components/PopupWithForm.js

```diff
@@ -1,68 +1,82 @@
 import Popup from "./Popup.js";
+import { createCard } from "../utils/createCard.js";
+import { handleSubmit } from "../utils/handleSubmit.js";
 
 export default class PopupWithForm extends Popup {
-  constructor(popupSelector, handleFormSubmit, handleFormClose = () => {}) {
+  constructor(popupSelector, handleSubmit, formValidator) {
     super({ popupSelector });
-    this._popup = document.querySelector(popupSelector);
-    this._handleFormSubmit = handleFormSubmit;
-    this._form = this._popup.querySelector("form");
-    this._inputs = Array.from(this._form.querySelectorAll("input"));
```

Comment:
` this._inputs = Array.from(this._form.querySelectorAll("input"));` were here. You shouldn't have changed it

Reply:
What you typed and line 9 of my code are exactly the same. I am confused as to what you want me to change here.